_posts/2020-07-12-ray-tune.md

```diff
@@ -0,0 +1,457 @@
+---
+layout: post
+title: Dask-ML and Ray have the same features for model selection
+tagline: Modern hyperparameter optimizations, Scikit-learn support, framework support and scaling to many machines.
+author: <a href="https://stsievert.com">Scott Sievert</a> (University of Wisconsin–Madison)
+tags: [machine-learning, dask-ml, dask, ray]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+Hyperparameter optimization is the process of deducing model parameters that
```

Comment:
Can this be omitted and the next paragraph be omitted and link to https://ml.dask.org/hyper-parameter-search.html instead? I think most people reading this will be familiar enough with hyperparameter optimization.

Reply:
Thanks, I've updated that paragraph.